pages/blog/_posts/reimagining-information-architecture.mdx

```diff
@@ -0,0 +1,82 @@
+
+![](/assets/blog/reimagining-information-architecture/image2.png)
+
+Next, we began to formulate hypotheses on users' current interactions with Inngest, and how they would desire these interactions to improve. What our users found valuable within Inngest, from a functionalist perspective, became more apparent and backed with quantitative data. In order to further validate these hypothese, we conducted user interviews with different user archetypes to gain qualitative insights, complementing our quantitative data.
+
+![](/assets/blog/reimagining-information-architecture/image3.png)
+We explored various different iterations of organizing information, as seen in the graphic above.
+
+We eventually landed on grouping navigation elements around the user's intent when visiting the Inngest dashboard: to Monitor and to Manage
+
+- **Monitor** gives the user tools to proactively and reactively debug, troubleshoot, and observe their systems built on Inngest. In the monitor grouping, you will find features such as the updated global runs view, and the in-progress observability dashboards.  
+- **Manage** allows the user to see and make changes to the structure and/or resources/entities of their system (Apps, Functions, Events, Webhooks) and is organized to reinforce users' mental model of Inngest. Additionally, we rethought the way that environments are displayed in order to give the user more clarity to which environment they are currently in. 
+
+![](/assets/blog/reimagining-information-architecture/image4.png)
+
+## How the new Information Architecture simplifies navigation
+
+### Enhance the overall developer experience through an intuitive navigation system
+
+By organizing information into two primary interaction modes, Monitor and Manage, we have created a navigation system that aligns closely with the natural workflow of our users. This intuitive structure allows developers to quickly find and use the tools they need, enhancing their overall experience and making their interactions with Inngest more efficient and satisfying.
+
+### Reduce the learning curve for users by ensuring consistency across products
+
+Our new IA offers consistent navigation and layout across both the Dev Server and Cloud Application. By standardizing the navigation and layout, we ensure that users can easily transition between these two environments without needing to relearn the interface. This consistency significantly reduces the learning curve and helps users become proficient more quickly. Before, users had difficulty switching between the Dev Server and Cloud Application. Aligning the functionality, layout, and terminology between the Dev Server and the Cloud Application reduces the cognitive load and for users, making transitioning between the Dev Server and the Cloud Application a much more seamless experience.  
```

Comment:
```suggestion
Our new IA offers consistent navigation and layout across both the Dev Server and Cloud Application. By standardizing the navigation and layout, we ensure that users can easily transition between these two environments without needing to relearn the interface. This consistency significantly reduces the learning curve and helps users become proficient more quickly. Before, users had difficulty switching between the Dev Server and Cloud Application because they differed in design. Aligning the functionality, layout, and terminology between the Dev Server and the Cloud Application reduced the cognitive load and made transitioning between the two environments a seamless experience for the users.
```